client/blocks/login/style.scss

```diff
@@ -1,6 +1,8 @@
+@import "@automattic/typography/styles/fonts";
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
 @import "@automattic/color-studio/dist/color-variables";
+@import "../../assets/stylesheets/shared/mixins/breakpoints";
```

Comment:
Storybook build of this individual file fails without an explicit import for this dependency